### PR TITLE
Added support for Innr RC110 Remote Control

### PR DESCRIPTION
**Pull request by danieledwardgeorgehitchcock**
Added support for Innr RC110 Remote Control

### CODE REVIEW

**Comment by danieledwardgeorgehitchcock** [comment]
Not sure why the check is failing? This is working locally...

**Comment by Koenkk** [comment]
Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/750

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -5777,6 +5777,14 @@ const converters = {
             return result;
         },
     },
+    rc_110_level_to_scene: {
```
Comment (Koenkk) [inline converters/fromZigbee.js:5780]: Why did you choose to add a custom converter instead of using `fz.command_move_to_level`?

Comment (danieledwardgeorgehitchcock) [reply converters/fromZigbee.js:5780]: The scenes buttons are reported as specific values and transition times which didn’t really make much sense from a user perspective (a pressed button will always be this value) as the buttons on the device are labelled as scenes, the returned brightness level isn’t really useful

Comment (Koenkk) [reply converters/fromZigbee.js:5780]: thanks!

**Comment by danieledwardgeorgehitchcock** [comment]
Resolves issues with #2255

**Pull request merged**